main.py

```diff
@@ -0,0 +1,128 @@
+
```

Comment:
helps to differentiate that file from others
when dealing with high numbers of python services, it's much easier when the entry point has a common name, like `main.py`
also helps for consistency as I've implemented:
```python
def main():
    print(assign_tasks())


if __name__ == '__main__':
    main()
```
`if __name__ == '__main__'` is a particularly important if statement if you have lots of python files within a project as it more or less defines an entry point into your program

similarly, the `def main()` bit is also pretty important as any variable declarations related to the startup of a script are contained within that function, otherwise they'd be globally accessible

just removes unnecessary risk of accidentally accessing duplicately named variable further down the line